lib/babylon-to-espree/attachComments.js

```diff
@@ -1,59 +0,0 @@
-"use strict";
-
-// comment fixes
-module.exports = function(ast, comments, tokens) {
```

Comment:
This was actually now incorrectly named. Rather than doing comment attachment, it's actually calculating the location information for the top-level `Program` node. Renamed this to `convertProgramNode` and encapsulated all the logic of transforming the `Program` node to what ESLint expects in that function.

Reply:
Agreed, much less confusing :)